2-mandatory/3-financial-times.js

```diff
@@ -4,36 +4,81 @@
     The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
-function potentialHeadlines(allArticleTitles) {
-    // TODO
+
+
+function potentialHeadlines(allArticleTitles){
+let shortTitles = [];
+let i = 0;
+
+while(i < allArticleTitles.length){
+    // console.log(ARTICLE_TITLES[i]);   
+    if (allArticleTitles[i].length < 66){
```

Comment:
While `< 66` produces the correct result in this case, it's good to get in the habit of encoding the requirement that contains the exact boundary -- for example, here, the articles need to be 65 characters or less -- so `<= 65` would be better -- this means you don't have to think whether what you're comparing against is an integer or a float (because, if the length was 65.5, `<= 65` would give you the correct result, but `< 66` would not).

Reply:
didn't think of it, thank you.